libs/stringtable/src/rapify.rs

```diff
@@ -0,0 +1,161 @@
+use crate::{Key, Project};
+use hemtt_workspace::WorkspacePath;
+use tracing::warn;
+
+static ALL_LANGUAGES: [&str; 14] = ["English", "Czech", "French", "Spanish", "Italian", "Polish", "Portuguese", "Russian", "German", "Korean", "Japanese", "Chinese", "Chinesesimp", "Turkish"];
```

Comment:
I would move this to the file where the struct is defined, maybe that file can be redone with codegen at some point to be cleaner